enterprise/management/commands/create_enterprise_course_enrollments.py

```diff
@@ -118,8 +119,8 @@ def create_enterprise_course_enrollments(self, course_id, enterprise_learners):
                     enterprise_customer_user=enterprise_learner,
                     course_id=course_id,
                 )
-
                 if created:
                     ent_course_enrollments_count += 1
+                    track_enrollment('management-command-enrollment', enterprise_learner.user_id, course_id)
```

Comment:
For some reason this one doesn't show up, and I'm not entirely sure why.

Reply:
Actually since this is a management command to handle missing enrollments, I'm wondering if we even need this? cc @douglashall @brittneyexline

Reply:
@UmanShahzad Agreed, I don't think we should fire an event here.

Reply:
Thanks, removed.